src/main/java/io/nats/client/api/KeyValueConfiguration.java

```diff
@@ -346,7 +346,7 @@ public KeyValueConfiguration build() {
                             .build());
                 }
             }
-            else if (sources.size() > 0) {
+            else if (!sources.isEmpty()) {
```

Comment:
Intellij hounds me about these. isEmpty checks the internal size variable. not sure why it always wants this, but it's the same amount of function calls